content/articles/phpunit-bigger-picture/index.md

```diff
@@ -0,0 +1,211 @@
+### Introduction
+
+Unit Tests in PHP or any other language are mainly written by developers as a good practice to help them identify and fix bugs in time, refactor code and serve as documentation for a unit of the program under test.
```

Comment:
Great work, start by removing all spaces beween the header and the first line of your paragraphs.